### ISSUE TITLE
Conversion of (0-5V) Analog Signal to (0-1.8V) without Loss of Information

### ISSUE THREAD

**Issue by NabilaTasnim** [beagleboard/beaglebone-black#20]
I have an analog signal of (0-5V). I want to use the ADC of BeagleBone Black to process it, but the ADC inputs can take a maximum of 1.8V. If I down-scale the signal to (0-1.8V), then -

How should I do this ensuring no loss/minimum loss of information or resolution? If there is some loss, then is there any way to retrieve the original signal after processing it with the ADC of BeagleBone Black?

**Comment by cschweers** [comment]
This is precisely the purpose of the so-called [voltage divider](https://en.wikipedia.org/wiki/Voltage_divider) in [electrical engineering](https://electronics.stackexchange.com/)

**Comment by NabilaTasnim** [comment]
If I use a voltage divider to scale down the voltage and then use ADC for digital conversion, won't I lose some accuracy and information? @cschweers @jadonk @RobertCNelson @pdp7

**Comment by jadonk** [comment]
yes, but not much depending on the accuracy of the resistors and amount of capacitance on the line.

**Comment by mvduin** [comment]
No, scaling the full range of the signal you want to measure to match the full range of the ADC you're using to measure it is exactly how you achieve the best accuracy possible with that particular ADC.

Also, this sort of question is better asked on the forum at https://forum.beagleboard.org/ instead of using a github issue.
